esp/src/src-react/components/Menu.tsx

```diff
@@ -78,16 +83,26 @@ interface DevMenuProps {
     location: string
 }
 
+const FIXED_WIDTH = 240;
+
 export const DevMenu: React.FunctionComponent<DevMenuProps> = ({
     location
 }) => {
 
-    const fixedWidth = 240;
+    const [favorites] = useFavorites();
+    const [menu, setMenu] = React.useState<INavLinkGroup[]>([...navLinkGroups]);
+
+    React.useEffect(() => {
+        navLinkGroups[0].links = Object.keys(favorites).map((key): INavLink => {
```

Comment:
Should this use a filter by name to find the appropriate group of links? They probably won't ever change position, but thought I'd mention it.

Reply:
Given the menu is currently just developers aid, its ok as is (for now)